integration/helpers/create-fixture.ts

```diff
@@ -174,6 +176,22 @@ export async function createFixtureProject(
       );
     }
   }
+
+  if (init.future) {
+    let contents = fse.readFileSync(
+      path.join(projectDir, "remix.config.js"),
+      "utf-8"
+    );
+    if (!contents.includes("future: {},")) {
+      throw new Error("Invalid formatted remix.config.js in template");
+    }
+    contents = contents.replace(
+      "future: {},",
+      "future: " + JSON.stringify(init.future) + ","
+    );
+    fse.writeFileSync(path.join(projectDir, "remix.config.js"), contents);
+  }
```

Comment:
If you pass a future config into `createFixtureProject` we'll write it into the remix config here